src/components/Constants/UseConstants.tsx

```diff
@@ -0,0 +1,6 @@
+import { useLoadConstantsQuery } from './LoadConstants.generated';
+
+// Use this call to force loading from cache storage first
+export const useApiConstants = useLoadConstantsQuery({
+  fetchPolicy: 'cache-first',
+});
```

Comment:
This is what I was alluding to in my last comment. I think it would be cleaner to return the constants object directly instead of making each consumer dig for it.
```suggestion
export const useApiConstants = () => {
  const { data } = useLoadConstantsQuery({
    fetchPolicy: 'cache-first',
  });

  return data?.constants;
};
```